### PR TITLE
 refactor: string and description improvements

### PR DESCRIPTION
**Pull request by Uberbagel**

### CODE REVIEW

File: mod_reforged/hooks/config/strings.nut [hunk 1]
```diff
@@ -1471,7 +1473,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 		}]
 	}),
 	RF_HaleAndHearty = ::UPD.getDescription({
-		Fluff = "Years of hard labor have given you a stamina like none other!",
+		Fluff = "\'Who\'s this chunk of meat?\'",
```
Comment (LordMidas) [inline mod_reforged/hooks/config/strings.nut:1476]: HaleAndHearty is unused in Reforged and should be removed.

File: mod_reforged/hooks/config/strings.nut [hunk 2]
```diff
@@ -219,7 +219,7 @@ local vanillaDescriptions = [
 		ID = "perk.bullseye",
 		Key = "Bullseye",
 		Description = ::UPD.getDescription({
-			Fluff = "Nailed it!",
+			Fluff = "One in the hand is worth two in the bush.",
```
Comment (LordMidas) [inline mod_reforged/hooks/config/strings.nut:222]: How about? `One through the gaps is worth two/many in the armor.`

**Review by LordMidas — CHANGES_REQUESTED** [review]

File: mod_reforged/hooks/config/strings.nut (hunk 2 above)
Comment (Uberbagel) [reply mod_reforged/hooks/config/strings.nut:222]: I like 'one in the hand' better

File: mod_reforged/hooks/config/strings.nut [hunk 3]
```diff
@@ -352,6 +353,7 @@ local vanillaDescriptions = [
 		ID = "perk.recover",
 		Key = "Recover",
 		Description = ::UPD.getDescription({
+			Fluff = "Better to rest for a moment than lose your head forever.",
```
Comment (LordMidas) [inline mod_reforged/hooks/config/strings.nut:356]: than `to` lose your head

**Review by LordMidas — CHANGES_REQUESTED** [review]

**Review by LordMidas — APPROVED** [review]

**Pull request merged**